README.md

```diff
@@ -27,6 +35,12 @@ If you want to install prerelease versions, you can do so by passing `--preview`
 python get-poetry.py --preview
```

Comment:
It's absolutely not clear from the docs, where to find this `get-poetry.py`, because `curl` (or `wget`) doesn't save it.

Reply:
@NickVolynkin

```console
wget https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
```

(Replace `curl -sSL` with just `wget`)

will save the install script to file.

Reply:
@hongquan Sure, it will. But documentation just supposes that the file is already there. It can confuse readers.

Reply:
@NickVolynkin Thanks. I will update that detail.